internal/observability/middleware_test.go

```diff
@@ -393,7 +418,7 @@ func TestUnaryInboundApplicationErrors(t *testing.T) {
 		zap.String("error", "application_error"),
 	}
 
-	core, logs := observer.New(zap.DebugLevel)
+	core, logs := observer.New(zap.ErrorLevel)
```

Comment:
I don't think that we should need to change this - we want this test spy to *collect* all logs (debug level and above), so that we're making assertions about the entire log output.